definitions/procedures/installer/run.rb

```diff
@@ -2,10 +2,13 @@ module Procedures::Installer
   class Run < ForemanMaintain::Procedure
     metadata do
       param :arguments, 'Arguments passed to installer'
+      param :assumeyes, 'Do not ask for confirmation'
     end
 
     def run
-      feature(:installer).run(@arguments, :interactive => true)
+      assumeyes_val = @assumeyes.nil? ? assumeyes? : @assumeyes
+      # If assumeyes selected we execute installer in non-interactive mode
```

Comment:
`feature(:installer).run(@arguments, :interactive => !assumeyes_val)`

This line explains everything from code perspective about interactive/non-interactive. So in my opinion, it would be good to include this information in the documentation not here.